ui/src/flux/components/TagListItem.tsx

```diff
@@ -83,46 +90,37 @@ export default class TagListItem extends PureComponent<Props, State> {
             </div>
           </CopyToClipboard>
         </div>
-        {this.state.isOpen && (
-          <>
-            <div
-              className="flux-schema--header"
-              onClick={this.handleInputClick}
-            >
-              <div className="flux-schema--filter">
-                <input
-                  className="form-control input-xs"
-                  placeholder={`Filter within ${tagKey}`}
-                  type="text"
-                  spellCheck={false}
-                  autoComplete="off"
-                  value={searchTerm}
-                  onChange={this.onSearch}
-                />
-                {this.isSearching && (
-                  <LoadingSpinner style={this.spinnerStyle} />
-                )}
-              </div>
-              {this.count}
+        <div className={`flux-schema--children ${isOpen ? '' : 'hidden'}`}>
```

Comment:
cool to see this idea. hidden instead of not rendering to preserve state. good thinking!